src/readonlyramdb.cpp

```diff
@@ -39,7 +39,7 @@ ReadonlyRamDb& ReadonlyRamDb::GetCached(const std::string& path,
                                         const std::size_t /*num_cu*/)
 {
     static std::mutex mutex;
-    static const std::lock_guard<std::mutex> lock{mutex};
+    std::lock_guard<std::mutex> lock{mutex};
```

Comment:
```suggestion
    const std::lock_guard<std::mutex> lock{mutex};
```

Hm, was the removal of the const needed?

Reply:
I do not think so. Fixed.